Description: Additional changes to work with the Debian package of singular
 We search for a specific SOVERSION of libsingular-Singular, to allow us to
 runtime-Depend on libsingular4 and not libsingular4-dev.
Author: Tobias Hansen <thansen@debian.org>
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage/interfaces/singular.py
+++ b/sage/src/sage/interfaces/singular.py
@@ -2271,7 +2271,7 @@
     nodes.clear()
     node_names.clear()
 
-    singular_docdir = SINGULARPATH + "/../info/"
+    singular_docdir = "/usr/share/doc/singular/"
 
     new_node = re.compile(r"File: singular\.hlp,  Node: ([^,]*),.*")
     new_lookup = re.compile(r"\* ([^:]*):*([^.]*)\..*")
--- a/sage/src/sage/misc/compat.py
+++ b/sage/src/sage/misc/compat.py
@@ -87,7 +87,7 @@
     EXAMPLES::
 
         sage: from sage.misc.compat import find_library
-        sage: find_library('Singular')
+        sage: find_library('singular-Singular')
         '...Singular...'
 
     """
--- a/sage/src/sage/env.py
+++ b/sage/src/sage/env.py
@@ -289,7 +289,7 @@
 
 # locate singular shared object
 # On Debian it's libsingular-Singular so try that as well
-SINGULAR_SO = _get_shared_lib_filename('Singular', 'singular-Singular')
+SINGULAR_SO = "/usr/lib/%s/libsingular-Singular-4.1.1.so" % sysconfig.get_config_var('MULTIARCH')
 var('SINGULAR_SO', SINGULAR_SO)
 
 # locate libgap shared object
